### PR TITLE
Change description when making Auth

### PR DESCRIPTION
**Pull request by PanRadaCloudBest**
Change description or conditions of unlocking scan QR when making Auth

### CODE REVIEW

File: src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx [hunk 1]
```diff
@@ -159,24 +160,34 @@ class ImportAuthenticatorScreen extends Component<Props, State> {
...
-              onPress={this.createAuthenticatorForm}
-            />
-            <FlatButton
-              containerStyle={styles.scanQRCodeButtonContainer}
-              title={i18n.wallets.importWallet.scanQrCode}
-              onPress={this.scanQRCode}
-              disabled={!name || !!this.validationError}
-            />
+            <TouchableWithoutFeedback onPress={this.sendFeedback}>
+              <Button
```
Comment (KedziaPawel) [inline src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx:177]: Do buttons still have touch indictor?

Comment (PanRadaCloudBest) [reply src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx:177]: is unblocked process

**Review by aleparty — APPROVED** [review]

**Review by KedziaPawel — APPROVED** [review]

File: src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx [hunk 2]
```diff
@@ -159,24 +160,34 @@ class ImportAuthenticatorScreen extends Component<Props, State> {
     return !!(name && mnemonic && !this.hasErrors());
   };
 
+  sendFeedback = () => {
+    const { name } = this.state;
+    if (!!!name.trim()) {
```
Comment (mateuszpmroz) [inline src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx:165]: I think that there's no difference between `!` and `!!!`

Comment (PanRadaCloudBest) [reply src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx:165]: yea id duplicate from code on top :)

**Pull request merged**